src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/DocFailureGenericException.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Event;
+
+public class DocFailureGenericException extends RuntimeException implements EventRelatedThrowable {
+
+    private static final Event EVENT = Event.DOC_FAILURE;
```

Comment:
I'd give it a more descriptive name than simply "EVENT"

Reply:
For public access makes sense. But I'll rename